eng/pipelines/templates/stages/python-analyze-weekly-standalone.yml

```diff
@@ -93,6 +93,20 @@ extends:
                   env:
                     GH_TOKEN: $(azuresdk-github-pat)
 
+                - task: PythonScript@0
+                  displayName: 'Get vNext Issues'
+                  continueOnError: true
+                  inputs:
+                    scriptPath: 'scripts/devops_tasks/dispatch_tox.py'
```

Comment:
Do we need a tox env for this task? Would a user run this locally? Should it just be a script?

Reply:
ah good point, I was using tox still to trigger pylint so that we could see if there were errors we needed to make an issue for

Reply:
they wouldnt run this locally